source/uri-options/tests/srv-service-name-option.yml

```diff
@@ -0,0 +1,16 @@
+tests:
+    - description: "SRV URI with custom srvServiceName"
+      uri: "mongodb+srv://test3.test.build.10gen.cc/?srvServiceName=customname"
+      valid: true
+      warning: false
```

Comment:
Why does this produce a warning?

Reply:
`~` (i.e. null) is used for things like `hosts` and `auth`, but I've only seen `true` and `false` used for `warning`. That agrees with https://github.com/mongodb/specifications/tree/master/source/uri-options/tests#format

Should this be `warning: false`? I think it was `warning: true` when I originally left a comment about this.

Reply:
It was originally `warning: false`, so I'll just switch back to that. It makes sense to assert the lack of warning in the success case.

Reply:
Oh ok, I totally misread the diff when I wrote my original comment. Sorry about that.